employees/templatetags/data_display_filters.py

```diff
@@ -0,0 +1,8 @@
+from django import template
+
+register = template.Library()
+
+
+@register.filter
+def decimal_to_hours(data):
+    return str(data).replace(".", ":")
```

Comment:
@Karrp @kbeker @maciejSamerdak 
By looking at this I have just realized we have a quite serious bug in the code.
We are holding working hours as decimal. It mean that when you create a report and set "0:30" (which I would understand as "half hour") in form, it stores value 0.30. When you set a value "0.5" (which I would as "half hour" too), you store value 0.50, but display 0:50.

So when I create a report and add "1:30" to my working hours, and another "1:30" in another report, how much to I get in total? Decimal 2.60.

This would be a serious issue when calculating sums etc. Now I am starting to understand why @maciejSamerdak was creating those complex transitions in him pull requests.

So we should ASAP change the field type of `work_hours` to `DurationField` https://docs.djangoproject.com/en/2.2/ref/models/fields/#durationfield. I have created an issue for that and I think it should have top priority.